src/index.js

```diff
@@ -1,39 +1,3 @@
+/* eslint-disable */
```

Comment:
- [ ] Kindly ensure you dont disable linter rules in your codebase because linters help to ensure that you write a bug-free, clean, and workable code. Always ensure you fix your linter errors instead of disabling them.